### DISCUSSION TITLE
Slave is not working with power share

### DISCUSSION THREAD

**Discussion by Smitje123** [dingo35/SmartEVSE-3.5#70] [General]
I have 2 smart EVSE chargers, if I use the setting no power sharing then the 2 chargers are working. If config master slave, then the master is working but the slave is not charging. Any tips what I do wrong?

**Comment by dingo35** [comment]
Post the /settings of both master and slave

**Reply by Smitje123** [reply]
Here the setting:

Master settings: (working)
{"version":"v3.6.0-RC1","serialnr":7198,"mode":"SMART","mode_id":3,"car_connected":false,"wifi":{"status":"WL_CONNECTED","ssid":"newwifi","rssi":-68,"bssid":"D2:32:E5:26:A0:1B"},"evse":{"temp":34,"temp_max":65,"connected":false,"access":true,"mode":1,"loadbl":1,"pwm":1024,"solar_stop_timer":0,"state":"Ready to Charge","state_id":0,"error":"None","error_id":0,"rfid":"Not Installed"},"settings":{"charge_current":150,"override_current":0,"current_min":6,"current_max":15,"current_main":20,"current_max_circuit":20,"current_max_sum_mains":600,"solar_max_import":0,"solar_start_current":4,"solar_stop_time":10,"enable_C2":"Not present","modem":"Not present","mains_meter":"Sensorbox","starttime":0,"stoptime":0,"repeat":0},"mqtt":{"host":"192.168.178.249","port":1883,"topic_prefix":"SmartEVSE-45453","username":"","password_set":false,"status":"Connected"},"home_battery":{"current":0,"last_update":0},"ev_meter":{"description":"Disabled","address":12,"import_active_power":0,"total_kwh":0,"charged_kwh":0,"currents":{"TOTAL":0,"L1":0,"L2":0,"L3":0},"import_active_energy":0,"export_active_energy":0},"mains_meter":{"import_active_energy":0,"export_active_energy":0},"phase_currents":{"TOTAL":-224,"L1":0,"L2":-165,"L3":-59,"last_data_update":1715102179,"original_data":{"TOTAL":-224,"L1":0,"L2":-165,"L3":-59}},"backlight":{"timer":0,"status":"OFF"}}

Slave settings: (not working)
{"version":"v3.6.0-RC1","serialnr":7199,"mode":"SMART","mode_id":3,"car_connected":false,"wifi":{"status":"WL_CONNECTED","ssid":"newwifi","rssi":-58,"bssid":"D2:32:E5:26:96:75"},"evse":{"temp":28,"temp_max":65,"connected":false,"access":true,"mode":1,"loadbl":2,"pwm":1024,"solar_stop_timer":0,"state":"Ready to Charge","state_id":0,"error":"None","error_id":0,"rfid":"Not Installed"},"settings":{"charge_current":0,"override_current":0,"current_min":6,"current_max":20,"current_main":20,"current_max_circuit":20,"current_max_sum_mains":600,"solar_max_import":0,"solar_start_current":4,"solar_stop_time":10,"enable_C2":"Not present","modem":"Not present","mains_meter":"Sensorbox","starttime":0,"stoptime":0,"repeat":0},"mqtt":{"host":"192.168.178.249","port":1883,"topic_prefix":"SmartEVSE-45433","username":"","password_set":false,"status":"Connected"},"home_battery":{"current":0,"last_update":0},"ev_meter":{"description":"Disabled","address":12,"import_active_power":0,"total_kwh":0,"charged_kwh":0,"currents":{"TOTAL":0,"L1":0,"L2":0,"L3":0},"import_active_energy":0,"export_active_energy":0},"mains_meter":{"import_active_energy":0,"export_active_energy":0},"phase_currents":{"TOTAL":-221,"L1":0,"L2":-164,"L3":-57,"last_data_update":1715102388,"original_data":{"TOTAL":-221,"L1":0,"L2":-164,"L3":-57}},"backlight":{"timer":58,"status":"ON"}}

Standalone settings: (working)
{"version":"v3.6.0-RC1","serialnr":7199,"mode":"SMART","mode_id":3,"car_connected":false,"wifi":{"status":"WL_CONNECTED","ssid":"newwifi","rssi":-62,"bssid":"D2:32:E5:26:96:75"},"evse":{"temp":28,"temp_max":65,"connected":false,"access":true,"mode":1,"loadbl":0,"pwm":1024,"solar_stop_timer":0,"state":"Ready to Charge","state_id":0,"error":"None","error_id":0,"rfid":"Not Installed"},"settings":{"charge_current":60,"override_current":0,"current_min":6,"current_max":15,"current_main":20,"current_max_circuit":20,"current_max_sum_mains":600,"solar_max_import":0,"solar_start_current":4,"solar_stop_time":10,"enable_C2":"Not present","modem":"Not present","mains_meter":"Sensorbox","starttime":0,"stoptime":0,"repeat":0},"mqtt":{"host":"192.168.178.249","port":1883,"topic_prefix":"SmartEVSE-45433","username":"","password_set":false,"status":"Connected"},"home_battery":{"current":0,"last_update":0},"ev_meter":{"description":"Disabled","address":12,"import_active_power":0,"total_kwh":0,"charged_kwh":0,"currents":{"TOTAL":0,"L1":0,"L2":0,"L3":0},"import_active_energy":0,"export_active_energy":0},"mains_meter":{"import_active_energy":0,"export_active_energy":0},"phase_currents":{"TOTAL":-213,"L1":0,"L2":-158,"L3":-55,"last_data_update":1715102241,"original_data":{"TOTAL":-213,"L1":0,"L2":-158,"L3":-55}},"backlight":{"timer":0,"status":"OFF"}}

Thanks

**Comment by dingo35** [comment]
Mainsmeter should be Disabled at the Slave; it gets its information from the 

EDIT: Master, not the Mainsmeter...

**Reply by Smitje123** [reply]
I change te value to disabled. Not working see see the pictures. 
![IMG_2507 2](https://github.com/dingo35/SmartEVSE-3.5/assets/169154073/81f9a6ab-f2e1-4fde-9d9d-6f333433358b)
![IMG_2508 2](https://github.com/dingo35/SmartEVSE-3.5/assets/169154073/baaa8ede-817a-48ba-801a-7ba6bd5d2064)
Then I see wrong current of -3218A ? In stand alone version current is the same as the master.

**Reply by Smitje123** [reply]
When I change the old hardware v2.2 to v3 I had this software.
![old-software](https://github.com/dingo35/SmartEVSE-3.5/assets/169154073/73632c2d-9c00-4044-bf56-1fa12a16d2cf)
And it works with master slave. Then I upgrade to v3.6 started to use MQTT. And the slave stop working. Is there a way to go back to the old software en clear all settings?

**Comment by dingo35** [comment]
So your Master, Slave and Sensorbox are all connected correctly to the same modbus?

**Reply by Smitje123** [reply]
Yes they are connected good, If I change number 1 as master or if I change number 2 as master it works. Do you have a link to the old software?

**Comment by dingo35** [comment]
Please downgrade both master and slave to v3.5.1

https://github.com/dingo35/SmartEVSE-3.5/releases/tag/v3.5.1

.

and report back here if your poblems still exist....

**Comment by dingo35** [comment]
Remember to flash both firmware.bin AND spiffs.bin!

**Comment by dingo35** [comment]
To clear all settings, after having flashed spiffs.bin, surf to 
http://ipaddress/erasesettings to erase your settings....

**Comment by dingo35** [comment]
Could you test this version, I think it solves your problem:

[4886937903a.zip](https://github.com/dingo35/SmartEVSE-3.5/files/15247880/4886937903a.zip)

**Reply by Smitje123** [reply]
Hi, This software works als master and slave. It is looking good. MQTT is also working. Thanks
